src/components/AccountInfo.tsx

```diff
@@ -91,34 +89,42 @@ export const AccountInfo = () => {
                       </>
                     }
                   >
-                    Create account
-                  </TextButton>
+                    <TextLink
+                      onClick={handleCreateAccount}
+                      disabled={account.status === ActionStatus.PENDING}
+                    >
+                      Create account
+                    </TextLink>
+                  </DetailsTooltip>
                 </div>
               )}
 
               {!account.isUnfunded && (
-                <TextButton
+                <TextLink
                   onClick={() =>
                     setIsAccountDetailsVisible(!isAccountDetailsVisible)
                   }
                 >{`${
                   isAccountDetailsVisible ? "Hide" : "Show"
-                } account details`}</TextButton>
+                } account details`}</TextLink>
               )}
             </div>
           </div>
 
           <div className="AccountInfoRow">
             <div className="AccountInfoCell">
               <div className="InfoButtonWrapper">
-                <TextButton
-                  onClick={handleRefreshAccount}
-                  disabled={account.status === ActionStatus.PENDING}
-                  tooltipText="If you performed account actions elsewhere, like in the
+                <DetailsTooltip
+                  details="If you performed account actions elsewhere, like in the
                   Stellar Laboratory, click here to update."
                 >
-                  Refresh account
-                </TextButton>
+                  <TextLink
+                    onClick={handleRefreshAccount}
+                    disabled={account.status === ActionStatus.PENDING}
```

Comment:
might be good to cache `account.status === ActionStatus.PENDING` as `isPending` since it's being checked twice